#goal definition
geometry_msgs/Point center
float64[3] normal
int32 primitive_type
int32 validate_type
---
#result definition
int32 result
---
#feedback
geometry_msgs/Point center
float64[3] normal
